### PR TITLE
Feature/DRP-5

### PR DESCRIPTION
**Pull request by Alturino**
# PURPOSES : 
Add Splash Screen UI
https://deuvox.atlassian.net/browse/DRP-5

# CHANGES : 
Add Image for splash screen
Add yellow color for background
Add splash screen layout
Add splash screen activity
Add splash screen component
Add splash screen module
Add splash screen view
Add splash screen view model
Add splash screen to android manifest

# PR NOTES : 
Please check the changes, are there any more improvements that can be made?

# TEST RESULT
None 🙅

### CODE REVIEW

File: app/src/main/java/com/deuvox/deuvoxapp/feature/main/splashscreen/SplashScreenView.kt [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : BaseRichView<SplashScreenBinding>(context, attrs, defStyleAttr) {
+
+    override fun getViewBinding(): SplashScreenBinding =
+        SplashScreenBinding.inflate(LayoutInflater.from(context), this, true)
+
+    override fun setup() {
+        binding.run {
+
```
Comment (ichsanachmad) [inline app/src/main/java/com/deuvox/deuvoxapp/feature/main/splashscreen/SplashScreenView.kt:20]: Please add todo if you want to add some action/statement in the future, but if is not, please remove the empty blocks

**Review by ichsanachmad — CHANGES_REQUESTED** [review]

**Review by ichsanachmad — CHANGES_REQUESTED** [review]
Please add test result (screenshot of UI that you made)

File: app/src/main/res/layout/splash_screen.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/yellow_primary">
+
+    <ImageView
+        android:id="@+id/splash_screen_image"
+        android:layout_width="320dp"
```
Comment (ichvandi) [inline app/src/main/res/layout/splash_screen.xml:10]: ini pake fixed size gini gpp ? ntar kalo deviceny kecil ato devicenya besar jd aneh gk ?

mungkin bisa pake rasio sm margin

**Review by ichvandi — COMMENTED** [review]
Ini yg png ganti ke xml aja biar gk perlu 4 image gini, kan di figma bisa export ke SVG 👍

**Comment by onirutlA** [comment]
> Ini yg png ganti ke xml aja biar gk perlu 4 image gini, kan di figma bisa export ke SVG 👍 

Sudah kak, tapi di android studio svg-ny gk kerender jdi pake png

File: app/src/main/res/layout/splash_screen.xml (hunk 2 above)
Comment (ichsanachmad) [reply app/src/main/res/layout/splash_screen.xml:10]: maybe you can ask @jeeshakira for this issue, beacuse it depends on her that we gonna use ratio or etc

**Comment by ichvandi** [comment]
> > Ini yg png ganti ke xml aja biar gk perlu 4 image gini, kan di figma bisa export ke SVG 👍
> 
> Sudah kak, tapi di android studio svg-ny gk kerender jdi pake png

coba kalo pake plugin ini bisa gak ?
https://www.figma.com/community/plugin/797369763563831541/Android-Vector-Drawable
+
https://www.figma.com/community/plugin/771155994770327940/Fill-Rule-Editor

**Comment by ichsanachmad** [comment]
Please read the suggestion from reviewer ya @onirutlA 👍

**Comment by Alturino** [comment]
> > > Ini yg png ganti ke xml aja biar gk perlu 4 image gini, kan di figma bisa export ke SVG 👍
> > 
> > 
> > Sudah kak, tapi di android studio svg-ny gk kerender jdi pake png
> 
> coba kalo pake plugin ini bisa gak ? https://www.figma.com/community/plugin/797369763563831541/Android-Vector-Drawable + https://www.figma.com/community/plugin/771155994770327940/Fill-Rule-Editor

sudah di coba kak itu gak ada gambarnya soalnya itu bukan vector, itu png kak jdi gak kerender gambarnya

**Comment by ichsanachmad** [comment]
> > > > Ini yg png ganti ke xml aja biar gk perlu 4 image gini, kan di figma bisa export ke SVG 👍
> > > 
> > > 
> > > Sudah kak, tapi di android studio svg-ny gk kerender jdi pake png
> > 
> > 
> > coba kalo pake plugin ini bisa gak ? https://www.figma.com/community/plugin/797369763563831541/Android-Vector-Drawable + https://www.figma.com/community/plugin/771155994770327940/Fill-Rule-Editor
> 
> sudah di coba kak itu gak ada gambarnya soalnya itu bukan vector, itu png kak jdi gak kerender gambarnya

i see, its an expected case bcs i got this problem too, but its alright then if we use png for a while, lets jump to another concern, have you fix another suggestion ?  @onirutlA if you already fixed it, please reopen this PR and notice me on telegram ya

**Pull request closed without merging**